src/content/docs/release-notes/mobile-apps-release-notes/new-relic-android-release-notes/new-relic-android-5400.mdx

```diff
@@ -0,0 +1,10 @@
+---
+subject: Mobile app for Android
+releaseDate: '2023-06-25'
+version: 5.5.0
+downloadLink: 'https://play.google.com/store/apps/details?id=com.newrelic.rpm'
+---
+
+### Notes
+
+Now with an updated Infrastructure experience!
```

Comment:
```suggestion
Updated the infrastructure monitoring experience.
```

Reply:
@bdwyer-newrelic Is this accurate, or is it referring to a specific infrastructure feature?

Reply:
Yep! That is accurate. The mobile app is always updating to reflect the web. This release was another instance of this.